modules/ROOT/nav.adoc

```diff
@@ -1,42 +1,42 @@
- ** xref:apikit-4-raml-text-task.adoc[Create a RAML Specification Using a Plain Text Editor]
- ** xref:start-project-task.adoc[To Start an APIkit Project]
- ** xref:run-apikit-task.adoc[To Run an APIkit Project]
- ** xref:apikit-simulate.adoc[To Simulate API Calls]
- ** xref:implement-endpoint-task.adoc[To Implement the Endpoint]
- ** xref:filter-users-id-task.adoc[To Filter Users By ID Using a Parameter]
- ** xref:add-names-service-task.adoc[To Add a Service to Get Only User Names]
- ** xref:test-api-task.adoc[To Test the API Implementation]
-* xref:apikit-4-tasks-index.adoc[Key APIkit 4.x Tasks]
- ** xref:validate-4-task.adoc[Validate Query Parameters and Headers Using APIKit]
- ** xref:regenerate-flows.adoc[To Change an APIkit Project]
- ** xref:customize-console-url-4-task.adoc[To Customize the API Console URL]
- ** xref:execute-nonfunctional-requirements-4-task.adoc[To Execute Non-functional Requirements]
- ** xref:configure-headers4-task.adoc[To Configure Headers]
- ** xref:update-4-task.adoc[To Update an APIkit Project]
- ** xref:retrieve-raml-task.adoc[To Retrieve RAML Files]
- ** xref:configure-cloudhub-path-task.adoc[To Configure the Path for Deployment on CloudHub]
- ** xref:configure-load-balancer-task.adoc[To Configure a Load Balancer]
-* xref:apikit-workflow-convert-existing.adoc[Workflow: Convert an Existing Mule Project to an APIkit Project]
-* xref:apikit-4-for-soap.adoc[Workflow: Implement a SOAP Interface]
- ** xref:apikit-4-soap-prerequisites-task.adoc[To Meet Prerequisites for Using APIkit for SOAP]
- ** xref:apikit-4-soap-project-task.adoc[To Create an APIkit for SOAP Project]
- ** xref:apikit-4-soap-fault-task.adoc[Identify Operations to Implement]
- ** xref:apikit-4-prevent-fault-task.adoc[To Prevent the SOAP Fault]
- ** xref:apikit-4-display-fault-task.adoc[To Display a Typed Fault]
- ** xref:apikit-4-get-header-task.adoc[To Get a SOAP Header]
- ** xref:apikit-4-set-header-task.adoc[To Set a SOAP Header]
-* xref:creating-an-apikit-4-project-with-maven.adoc[Creating an APIkit Project with Maven]
-* xref:apikit-retrieve-raml.adoc[Retrieving RAML Resources Reference]
-* xref:handle-errors-4-concept.adoc[About Handling Errors]
-* xref:apikit-error-handling-reference.adoc[APIkit Error Handling Reference]
-* xref:apikit-4-xml-reference.adoc[APIkit 4.x XML Reference]
-* xref:apikit-4-for-odata.adoc[APIkit for OData]
- ** xref:creating-an-odata-api-with-apikit.adoc[Create an OData v2 API with APIkit]
- ** xref:apikit-odata-extension-reference.adoc[APIkit OData v2 Extension Reference]
- ** xref:creating-an-odatav4-api-with-apikit.adoc[Create an OData v4 API with APIkit]
-  *** xref:apikit-odatav4-expand-feature.adoc[Implement the $expand Feature]
- ** xref:apikit-odatav4-extension-reference.adoc[APIkit OData v4 Extension Reference]
\ No newline at end of file
+.xref:index.adoc[APIkit]
+* xref:index.adoc[Overview]
+ ** xref:apikit-compatibility.adoc[APIkit Compatibility]
```

Comment:
There shouldn't be a single bullet.

Reply:
If you look at the compatibility page, you'll notice some relevant information about the modules, the language, and the protocols APIkit supports. Should we move this info to the RNs?

Reply:
The compatibility information is displayed not only on this page but also in the release notes.
This topic has been discussed with Cris V and he approved this structure.
Thanks!